### PR TITLE
To disable WebNotification for Crosswalk for Android

### PR DESCRIPTION
**Pull request by PeterWangIntel**
Building with "-Ddisable_notifications=1" reduces size by 68K.

The 'notifications' is an independent module in 'modules' folder.

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@d0ngjun
It's the clean patch cherry-picked to crosswalk-lite branch.

File: Source/modules/modules_gypi_generator.py [hunk 1]
```diff
@@ -449,9 +482,15 @@ def modules_dictionary_idl_files():
...
       'mediastream/RTCSessionDescriptionInit.idl',
+    """
+
+    str_notifications = """
       'notifications/GetNotificationOptions.idl',
       'notifications/NotificationEventInit.idl',
       'notifications/NotificationOptions.idl',
+    """
+
+    str_1 = """
```
Comment (halton) [inline Source/modules/modules_gypi_generator.py]: Same comment, please use meaningful variable.

Comment (halton) [inline Source/modules/modules_gypi_generator.py]: @PeterWangIntel any follow up?

**Comment by PeterWangIntel** [comment]
@d0ngjun

File: Source/modules/modules_gypi_generator.py (hunk 1 above)
Comment (PeterWangIntel) [inline Source/modules/modules_gypi_generator.py]: It's been disscussed face to face already.

**Comment by PeterWangIntel** [comment]
@halton,  please merge this patch, since the corespondent part in 'chromium' is merged already.

**Pull request merged**